packages/react-charting/src/components/Legends/Legends.base.tsx

```diff
@@ -49,11 +49,14 @@ export class LegendsBase extends React.Component<ILegendsProps, ILegendState> {
 
   public constructor(props: ILegendsProps) {
     super(props);
+    const defaultSelectedLegends =
+      props.defaultSelectedLegends?.reduce((combinedDict, key) => ({ [key]: true, ...combinedDict }), {}) || {};
```

Comment:
@krkshitij to check if there is any corner case missing in the implementation here.

Reply:
Looks good